azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachine.java

```diff
@@ -237,15 +239,15 @@
      * @return the refreshed instance view
      */
     @Method
-    VirtualMachineInstanceView refreshInstanceView();
+    VirtualMachineInstanceViewInner refreshInstanceView();
```

Comment:
We should not return Inner to the user.
If AutoRest changes this type to Inner, usually means this type is now being returned by an API directly. You will have to wrap it with an interface and add an impl.

Reply:
Bingo: https://github.com/Azure/azure-rest-api-specs/blob/78e1feab142c605839aa3cec23442ca26b8ef04d/specification/compute/resource-manager/Microsoft.Compute/stable/2018-10-01/compute.json#L1753

@anuchandy

Reply:
done